src/CRM/CivixBundle/Resources/views/Code/test-headless.php.php

```diff
@@ -28,34 +29,43 @@
  */
 class <?php echo $testClass ?> extends \PHPUnit\Framework\TestCase implements HeadlessInterface, HookInterface, TransactionalInterface {
 
-  public function setUpHeadless() {
-    // Civi\Test has many helpers, like install(), uninstall(), sql(), and sqlFile().
-    // See: https://docs.civicrm.org/dev/en/latest/testing/phpunit/#civitest
+  /**
+   * Setup used when HeadlessInterface is implemented.
+   * 
+   * Civi\Test has many helpers, like install(), uninstall(), sql(), and sqlFile().
+   * 
+   * @see See: https://github.com/civicrm/org.civicrm.testapalooza/blob/master/civi-test.md
+   *
+   * @return \Civi\Test\CiviEnvBuilder
+   *
+   * @throws \CRM_Extension_Exception_ParseException
+   */
+  public function setUpHeadless():void {
```

Comment:
not sure if this is needed as its a internal Civi only thing but probably good to do

Reply:
Doesn't make sense on a few fronts:

* The function here actually does return a value, which raises a PHP error.
* This disagrees with `HeadlessInterface`'s signature for `setUpHeadless()`
* Because the templates have it return a value... therefore one should expect extant tests to also return a value.

Reply:
updated the return to   ```CiviEnvBuilder```